src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -583,6 +583,13 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode)
                         ->end()
                         ->booleanNode('logging')->defaultValue($this->debug)->end()
                     ->end()
+                    ->fixXmlConfig('path')
+                    ->children()
+                        ->arrayNode('paths')
+                            ->prototype('scalar')->end()
+                            ->defaultValue(array())
+                        ->end()
+                    ->end()
```

Comment:
What about merging this node and the `fallbacks` node into one single `children()` block?